wgsl/index.bs

```diff
@@ -528,14 +528,22 @@ An <dfn>identifier</dfn> is a kind of [=token=] used as a name.
 See [[#declaration-and-scope]] and [[#directives]].
 
 The form of an identifier is defined via pattern-matching, except that
-an identifier must not have the same spelling as a [=keyword=] or as a [=reserved word=].
+an identifier:
+* must not have the same spelling as a [=keyword=] or as a [=reserved word=].
+* must not be `_` (a single underscore)
+* must not start with two undercores.
 
 <div class='syntax' noexport='true'>
   <dfn for=syntax>ident</dfn> :
 
-    | `/[a-zA-Z][0-9a-zA-Z_]*/`
+    | `/[a-zA-Z]([0-9a-zA-Z][0-9a-zA-Z_]*)?/`
```

Comment:
This regex might be not acceptive of `a_b` kind of identifiers, which, if I am not mistaken, should be allowed.

Reply:
Right. I see that.  Whoops.

Reply:
Thanks for the fix, the regex looks exactly like my first imagination right now 😆